.env.example

```diff
@@ -1,3 +1,5 @@
+# Please see https://github.com/theopensystemslab/planx-new/blob/main/doc/architecture/decisions/0004-secret-generation.md for procedure on secrets generation
```

Comment:
Usually one would reference the file locally (e.g. `doc/architecture/…`) as opposed to linking to GitHub (e.g. `https://…`) but this is fine :+1: